### PR TITLE
Clarify relationship between CR and CG report

### LINKED ISSUE #436: Clarify SOTD and abstract of proposed CR
Clarify that the CR is based on the CG report, and that it is a product of the TTWG.

### PR DESCRIPTION
**Pull request by palemieux**
Closes #436

### CODE REVIEW

**Comment by silviapfeiffer** [comment]
Can we get @tmichel07 input on this? What is the usual way this is done in the W3C?

**Comment by nigelmegitt** [comment]
> not as clear about the relationship as I would like, but whatever

@dwsinger What aspects of the relationship do readers of the spec need to know about?

File: archives/2018-04-15/Overview.html [hunk 1]
```diff
@@ -1571,13 +1567,6 @@ <h2 class="no-num no-toc no-ref heading settled" id="status"><span class="conten
 <p> This document is governed by the <a href="https://www.w3.org/2017/Process-20180201/" id="w3c_process_revision">1 February 2018 W3C Process Document</a>.</p>
 
   </div>
-  <div class="snapshot" hidden="">
-   <p>
-    This is a Recommendation Track snapshot. For the latest updates, possibly including important
- bug fixes, please see the <a href="https://w3c.github.io/webvtt/">Draft Community Group Report</a>. 
-    <button>Dismiss</button>
```
Comment (nigelmegitt) [inline archives/2018-04-15/Overview.html]: Not sure why the snapshot guidance needs to be removed; I agree with @silviapfeiffer we need W3 guidance on what is "normal" when managing specs in two places at once like this.

Comment (nigelmegitt) [reply archives/2018-04-15/Overview.html]: That's true. Plus I think the best forum for communicating about the spec is via GitHub repo issues, and anyone can do that whether they're in the CG, WG, both or neither.

Comment (silviapfeiffer) [reply archives/2018-04-15/Overview.html]: I propose we change
&lt;a href="https://w3c.github.io/webvtt/">Draft Community Group Report&lt;/a>
to
&lt;a href="https://w3c.github.io/webvtt/">Editor's Draft&lt;/a>

Comment (dwsinger) [reply archives/2018-04-15/Overview.html]: OK by me

**Comment by dwsinger** [comment]
I think Silvia's change is fine. Let's do it. Getting the CR done is more important than finessing this more.

File: archives/2018-04-15/Overview.html (hunk 1 above)
Comment (nigelmegitt) [reply archives/2018-04-15/Overview.html]: There should already be an Editor's Draft link at the top of the document. If the solution is to change the link to the ED then we should just remove the whole paragraph as per the pull request. Folk don't need to be told that the ED might be more up to date, that's obvious.

**Comment by silviapfeiffer** [comment]
OK, let's merge it then.

**Pull request merged**